packages/meditrak-server/src/database/models/UserReward.js

```diff
@@ -56,6 +56,7 @@ export class UserRewardModel extends DatabaseModel {
       "'5ea3984a61f76a61f1011fa7'", // Geoff F
       "'5cee0ba0f013d666713786a9'", // mSupply API Client
```

Comment:
Could you check if any of the above can be removed, i.e. if they have an internal email?

Reply:
I can confirm that all of them are not bes emails 👍